tripy/tests/backend/test_compiler_api.py

```diff
@@ -62,6 +62,14 @@ class TestInput:
             ([(1, 2, 3)], (1,), (2,), (3,)),
             # Only one value specified
             ([1], (1,), (1,), (1,)),
+            # one dynamic and one static dim
+            ([(1, 2, 3), 4], (1, 4), (2, 4), (3, 4)),
+            # Both dim dynamic
+            ([(1, 2, 3), (4, 5, 6)], (1, 4), (2, 5), (3, 6)),
+            # min/opt/max specified as shape tensor
+            ([tp.Shape([1, 2, 3])], (1,), (2,), (3,)),
```

Comment:
@pranavm-nvidia this is how you expected shape profile to be provided using shape tensor?